app.py

```diff
@@ -147,29 +152,44 @@
 app = App(token=os.environ.get("SLACK_BOT_TOKEN"), signing_secret=os.environ.get("SLACK_SIGNING_SECRET"))
 
 
+def get_channel_bot(channel: str):
```

Comment:
If we want to be fancier, we could store them in a dict to avoid this whole function

Reply:
indeed... would be much cleaner